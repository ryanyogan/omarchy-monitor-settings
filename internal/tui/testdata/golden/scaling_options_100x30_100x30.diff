Visual regression detected!

=== EXPECTED ===
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m│[0m                                      [1;34mDisplay Settings[0m                                      [90m│[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                    
  [32m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [1;32m🧠 Smart Scaling Recommendations[0m                                                          [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [33m╭─────────────────────────────────────────╮[0m                                               [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                                               [32m│[0m    
  [32m│[0m  [33m│[0m[40m  [0m[40mHDMI-A-1 Dell U2414H - 1920x1080@60Hz[0m[40m  [0m[33m│[0m                                               [32m│[0m    
  [32m│[0m  [33m│[0m[40m                                         [0m[33m│[0m                                               [32m│[0m    
  [32m│[0m  [33m╰─────────────────────────────────────────╯[0m                                               [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [1;36m🎯 Available Options[0m                                                                      [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [1;34m▶ [0m[1;34m1x Native [RECOMMENDED][0m                                                                 [32m│[0m    
  [32m│[0m  [37m    Native resolution with standard scaling[0m                                               [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080[0m                            [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                   [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m    1.25x Enhanced                                                                          [32m│[0m    
  [32m│[0m  [37m    Slightly larger text for better readability[0m                                           [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864[0m                            [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                   [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m    1.5x Large                                                                              [32m│[0m    
  [32m│[0m  [37m    Larger text for accessibility[0m                                                         [32m│[0m    
  [32m│[0m  [90m    Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720[0m                            [32m│[0m    
  [32m│[0m  [3;90m    💡 [0m                                                                                   [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [1;35m📚 What Each Setting Does[0m                                                                 [32m│[0m    
  [32m│[0m  [37m  Monitor Scale: Changes compositor-level scaling (immediate effect)[0m                      [32m│[0m    
  [32m│[0m  [37m  GTK Scale: Scales GTK applications (requires logout/login)[0m                              [32m│[0m    
  [32m│[0m  [37m  Font DPI: Fine-grained text scaling (affects most apps)[0m                                 [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m  [32m↑↓[0m[37m select[0m  [34m⏎[0m[37m apply[0m  [33mm[0m[37m manual[0m  [35mesc[0m[37m back[0m                                                    [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m│[0m                                                                                            [32m│[0m    
  [32m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    
                                                                                                    
  [90m╭────────────────────────────────────────────────────────────────────────────────────────────╮[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m│[0m                 [1;32m↑↓[0m  [37mnavigate[0m    [1;34m⏎[0m  [37mselect[0m    [1;33mh[0m  [37mhelp[0m    [1;35mesc[0m  [37mback[0m    [1;31mq[0m  [37mquit[0m               [90m│[0m    
  [90m│[0m                                                                                            [90m│[0m    
  [90m╰────────────────────────────────────────────────────────────────────────────────────────────╯[0m    

=== ACTUAL ===
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                      Display Settings                                      │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                                                                            │    
  │  🧠 Smart Scaling Recommendations                                                          │    
  │                                                                                            │    
  │  ╭─────────────────────────────────────────╮                                               │    
  │  │                                         │                                               │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz  │                                               │    
  │  │                                         │                                               │    
  │  ╰─────────────────────────────────────────╯                                               │    
  │                                                                                            │    
  │  🎯 Available Options                                                                      │    
  │                                                                                            │    
  │  ▶ 1x Native [RECOMMENDED]                                                                 │    
  │      Native resolution with standard scaling                                               │    
  │      Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │    1.25x Enhanced                                                                          │    
  │      Slightly larger text for better readability                                           │    
  │      Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │    1.5x Large                                                                              │    
  │      Larger text for accessibility                                                         │    
  │      Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │                                                                                            │    
  │  📚 What Each Setting Does                                                                 │    
  │    Monitor Scale: Changes compositor-level scaling (immediate effect)                      │    
  │    GTK Scale: Scales GTK applications (requires logout/login)                              │    
  │    Font DPI: Fine-grained text scaling (affects most apps)                                 │    
  │                                                                                            │    
  │  ↑↓ select  ⏎ apply  m manual  esc back                                                    │    
  │                                                                                            │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                 ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit               │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    

=== DIFF INFO ===
Expected length: 7852
Actual length: 6573
Expected hash: d0bdf99a45f4ffb553b9410a16d2981e2100b0f9bcf8c86467ba020bbd8f3079
Actual hash: d2031caa95296ab3863c01ecd585d78348ddf5ac73de03be9691364ce9335449
