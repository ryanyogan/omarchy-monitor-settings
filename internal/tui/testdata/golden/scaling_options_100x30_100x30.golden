# Visual Golden File
# Name: scaling_options_100x30
# Dimensions: 100x30
# Hash: d2031caa95296ab3863c01ecd585d78348ddf5ac73de03be9691364ce9335449

  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                      Display Settings                                      │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                                                                                            │    
  │  🧠 Smart Scaling Recommendations                                                          │    
  │                                                                                            │    
  │  ╭─────────────────────────────────────────╮                                               │    
  │  │                                         │                                               │    
  │  │  HDMI-A-1 Dell U2414H - 1920x1080@60Hz  │                                               │    
  │  │                                         │                                               │    
  │  ╰─────────────────────────────────────────╯                                               │    
  │                                                                                            │    
  │  🎯 Available Options                                                                      │    
  │                                                                                            │    
  │  ▶ 1x Native [RECOMMENDED]                                                                 │    
  │      Native resolution with standard scaling                                               │    
  │      Monitor: 1.0x • GTK: 1x • Font DPI: 96 • Result: 1920x1080                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │    1.25x Enhanced                                                                          │    
  │      Slightly larger text for better readability                                           │    
  │      Monitor: 1.2x • GTK: 1x • Font DPI: 120 • Result: 1536x864                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │    1.5x Large                                                                              │    
  │      Larger text for accessibility                                                         │    
  │      Monitor: 1.5x • GTK: 1x • Font DPI: 144 • Result: 1280x720                            │    
  │      💡                                                                                    │    
  │                                                                                            │    
  │                                                                                            │    
  │  📚 What Each Setting Does                                                                 │    
  │    Monitor Scale: Changes compositor-level scaling (immediate effect)                      │    
  │    GTK Scale: Scales GTK applications (requires logout/login)                              │    
  │    Font DPI: Fine-grained text scaling (affects most apps)                                 │    
  │                                                                                            │    
  │  ↑↓ select  ⏎ apply  m manual  esc back                                                    │    
  │                                                                                            │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    
                                                                                                    
  ╭────────────────────────────────────────────────────────────────────────────────────────────╮    
  │                                                                                            │    
  │                 ↑↓  navigate    ⏎  select    h  help    esc  back    q  quit               │    
  │                                                                                            │    
  ╰────────────────────────────────────────────────────────────────────────────────────────────╯    